cpp/src/gandiva/like_holder.h

```diff
@@ -1,62 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Just to confirm, this code can really be removed right?

Reply:
I renamed the file to `string_functions_holder.cc`, I think Github is not displaying it.